requirements.md

```diff
@@ -16,29 +16,46 @@ See also: [SPC Scope](scope.md) for definitions and more information.
 ## Web Context Support
 
 * It must possible to invoke SPC from an iframe. See [issue 68](https://github.com/w3c/secure-payment-confirmation/issues/68).
+
+Q: I understand that woudl also require PR API to be used from within the IFrame. Is that right?
```

Comment:
If SPC is ultimately spec'd in terms of the PR API, yes. The explainer/experimental version of SPC was done that way (`new PaymentRequest([{supportedMethods: 'secure-payment-confirmation', ...}], ...).show()`), but this doesn't mean we _have_ to write the 'real' spec the same way (though some of us are biased in that direction ;)).

Do you have a concern regarding 'PR API to be used from within the IFrame'?

Reply:
Not really. I just wanted to be clear that the invocation of SPC auth from within the IFrame would be the same as from the main context. I think we're on the same page.

Reply:
Hearing "same page" and having updated the doc, I'm going to close this part of the conversation.